src/NeuralNetDiffEq.jl

```diff
@@ -60,19 +60,19 @@ function Base.show(io::IO, A::KolmogorovPDEProblem)
   show(io , A.g)
 end
 
-struct NNPDEProblem{PDESystem,MOLFiniteDifference,P} <:DiffEqBase.DEProblem
+struct discretize{PDESystem,PhysicsInformedNN,P} <:DiffEqBase.DEProblem
```

Comment:
discretize is a function in DiffEqBase that you should be extending.

Reply:
ok